.github/workflows/linux_debug.yml

```diff
@@ -19,7 +19,7 @@ env:
 
 jobs:
   build:
-    runs-on:  ubuntu-18.04
+    runs-on:  ubuntu-20.04
```

Comment:
So the fact that the Linux versions are built on Ubuntu 18 is important since it allows pre-built versions of QGC to run on Ubuntu 18 and 20. Moving to 20 will lose that capability. Maybe we just toss support for 18 with this newest version of QGC. I think 18 is getting pretty old now right?

Reply:
So I assume Ubuntu 20 comes with gstreamer 1.20 right?

Reply:
Wait just say the note about Ubuntu 22. How can that be feasible then? We are going to need Ubuntu 20 support I would say since that is still pretty recent.

Reply:
Yeah I agree with you, I think bumping up to Ubuntu 20.04 is good, but we shouldn't move our ci stack to 22 for a while.

I'll make a separate PR bumping our ubuntu from 18 to 20 and then I'll update this PR with a manual install of gstreamer in the ci if that works?

Reply:
That would also mean that the AppImage needs to include the updated version of gstreamer as well so that it just works out of the box without additional installs. Doesn't it?

Reply:
Also I have a quick question: if the CI is building the AppImage then does it matter which Ubuntu version we base off of? Does the AppImage contain everything that is needed to run qgroundcontrol?

Reply:
"if the CI is building the AppImage then does it matter which Ubuntu version we base off of?" You would think so. But something was linking to a specific OS lib which didn't exist or maybe different version on 20. I think there is a closed issue on it somewhere. I think @mrpollo  discovered the problem. You can probably go through commit blame on the CI file to find the change which was recent.

Reply:
Weird I guess TIL.

Reply:
@mrpollo Do you remember if there an issue on the Ubuntu 18 problem?

Reply:
Yeah it was a `glibc` version incompatibility see #10016, the only viable alternative at the time was to downgrade to ubuntu-18, especially since we didn't really have a version to upgrade to ubuntu-20 in the first place

do we have a reason to upgrade now?

Reply:
Not really the version that this PR needs is Ubuntu 22 or building gstreamer from source. I've been trying to get gstreamer building from source for now as Ubuntu 22 is too far of a platform to target. Updating to Ubuntu 20 will not help this PR.

Reply:
Given that we should keep CI at 18 then to keep back compat.